sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/BlobsDeleteImmutabilityPolicyHeaders.java

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import java.time.OffsetDateTime;
+
+/** The BlobsDeleteImmutabilityPolicyHeaders model. */
+@JacksonXmlRootElement(localName = "null")
```

Comment:
Same question here @srnagar, @jianghaolu

Reply:
I've seen other model types in the storage libraries with null set here and it hasnt caused any problems so far. This is a class to deserialize headers so it should be fine right?